src/styles/card/card.scss

```diff
@@ -0,0 +1,113 @@
+.main {
+  padding: 50px 40px;
+  height: 862px;
+  max-width: 1200px;
+
+  /* Центрування елемента горизонтально */
+  margin: 0 auto;
```

Comment:
It's better to group related properties together. Margin property should be placed near padding property as they both are box model properties.